scala/build.sbt

```diff
@@ -55,13 +55,12 @@ pomExtra := (
 )
 
 libraryDependencies ++= Seq(
-  "com.google.guava" % "guava" % "21.0",
-  "com.jsuereth" %% "scala-arm" % "2.0",
```

Comment:
What was that and why isn't it needed anymore?

Reply:
Automatic Resource Management. Scala 2.13 has that in the core library (the whole Releasable/Using stuff), compare https://github.com/jsuereth/scala-arm/issues/79